platforms-whitepaper/v1alpha1/paper.md

```diff
@@ -153,6 +162,7 @@ for their users to deliver valuable products.
 [1]: https://www.atlassian.com/devops/frameworks/team-topologies
 [2]: https://martinfowler.com/articles/talk-about-platforms.html
 [3]: https://teamtopologies.com/key-concepts-content/what-is-a-thinnest-viable-platform-tvp
+[4]: https://thenewstack.io/architecture-and-design-considerations-for-platform-engineering-teams/
```

Comment:
It's okay not to have a reference for this IMO, can we take it out or move to the bottom?